pkg/lib/cockpit-components-inline-notification.tsx

```diff
@@ -20,57 +20,57 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import cockpit from 'cockpit';
 
-import { Alert, AlertActionCloseButton } from "@patternfly/react-core/dist/esm/components/Alert/index.js";
+import { Alert, AlertActionCloseButton, AlertProps } from "@patternfly/react-core/dist/esm/components/Alert/index.js";
```

Comment:
This is more TIL a few days ago, you can import this as:

`import type AlertProps` so it won't end up in bundled JS.

Reply:
Done, thanks! I fixed the other instance in pkg/lib/cockpit-components-dropdown.tsx as well.